Main.java

```diff
@@ -1,5 +1,8 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("hello");
+        int[] numbers = {1, 2, 3};
```

Comment:
Consider using a more descriptive method name instead of 'multiply' to clearly indicate its purpose.